[WPI]DB設計

yukri1030 opened the pull request:
## WHAT
DB設計に必要なテーブル、カラム、アソシエーションを作りました

## WHY
しっかりとしたDB設計が、スムーズなアプリ開発に繋がるためです

yukri1030 committed:
[WPI]DB設計

## WHAT
それそれのテーブルとカラムを作りました

## WHY
足りない部分があれば、よろしくお願いします

TOMARUMARU commented on README.md:
```diff
@@ -0,0 +1,57 @@
+|Column|Type|Opitions|
+|------|----|---------|
+|name|string|null: false,uniqe:true|
+|messages_id|reference|null: false,foreign_key :true|
```
messageのidは不要ですね

yukri1030 replied:
削除しました

TOMARUMARU commented on README.md:
```diff
@@ -0,0 +1,57 @@
+|Column|Type|Opitions|
+|------|----|---------|
+|name|string|null: false,uniqe:true|
+|messages_id|reference|null: false,foreign_key :true|
```
こちらも

yukri1030 replied:
削除しました

TOMARUMARU commented on README.md:
```diff
@@ -0,0 +1,57 @@
+|------|----|---------|
+|user_id|reference|null: false,foreign_key :true|
+|group_id|reference||null:false, foreign_key: true|
+|message_id|reference|null: false,foreign_key :true|
```
こちらも

yukri1030 replied:
削除しました

TOMARUMARU commented on README.md:
```diff
@@ -0,0 +1,57 @@
+
+### Asociation
+- has_many: users through: members
+- has_many: messages through: users
```
has_many: messages through: usersというのはおかしいですね

yukri1030 replied:
has_many: messages through: membersに変更しました！

yukri1030 committed:
DB見直し

# WHAT
message_id　は削除しました
groupテーブルのアソシエーションを以下のように変更しました
has_many: messages through: members

# WHY
スムーズなアプリ開発のため、DB設計しました。

ken-nag commented:
lgtm
